.github/workflows/release.yml

```diff
@@ -6,20 +6,22 @@ on:
       - 'beta'
       - 'alpha'
   pull_request: {}
+
+# Cancel any previous run (see: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#concurrency)
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
 jobs:
   main:
     runs-on: ubuntu-latest
     steps:
-      - name: Cancel Previous Runs
-        uses: styfle/cancel-workflow-action@0.6.0
-        with:
-          access_token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
As recommended by https://github.com/styfle/cancel-workflow-action#readme:
> **Warning**
> You probably don't need to install this custom action.
> 
> Instead, use the native [concurrency](https://github.blog/changelog/2021-04-19-github-actions-limit-workflow-run-or-job-concurrency/) property to cancel workflows, for example:
> ```
> concurrency:
>   group: ${{ github.workflow }}-${{ github.ref }}
>   cancel-in-progress: ${{ github.ref != 'refs/heads/main' }}
> ```
> Read GitHub's [official documentation](https://docs.github.com/en/actions/using-jobs/using-concurrency) to learn more.

=> replacing with native github workflow `concurrency:` https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#concurrency

Reply:
I leave the `${{ github.ref != 'refs/heads/master' }}` apart, since I don't see any inconvenient not cancelling the master branch also (if another more recent push), but you tell me